site/docs/v1/tech/events-webhooks/events.adoc

```diff
@@ -27,6 +27,24 @@ These are the events that FusionAuth generates that can be optionally consumed b
 * <<JWT Refresh>> - when an access token is refreshed using a refresh token
 * <<JWT Refresh Token Revoke>> - when a refresh token (or multiple tokens) are revoked
 
+=== Application Scoped Events
+
+Some events can be generated for one or more specified applications. These events may also be generated for all applications within a tenant.
+
+The events that are considered "application" events and can be scoped to an application based up the Webhook config: 
+
+* `user.action`
+* `jwt.refresh-token.revoke`
+* `jwt.refresh`
+* `jwt.public-key.update`
+
+=== Tenant Scoped Events
+
+Any events which are not application scoped events are by tenant scoped. 
```

Comment:
> ... are by default tenant scoped

Assuming you're missing a `default` in that sentence.

Reply:
Yup, doh!